spring-integration-ftp/src/test/java/org/springframework/integration/ftp/outbound/FtpServerOutboundTests-context.xml

```diff
@@ -53,6 +53,7 @@
 							  command="mget"
 							  expression="payload"
 							  command-options="-R -P"
+							  mode="REPLACE_IF_DIFFERENT"
```

Comment:
??? `REPLACE_IF_MODIFIED` ?
Otherwise:
```
Caused by: java.lang.IllegalStateException: Cannot convert value of type 'java.lang.String' to required type 'org.springframework.integration.file.support.FileExistsMode' for property 'fileExistsMode': no matching editors or conversion strategy found
	at org.springframework.beans.TypeConverterDelegate.convertIfNecessary(TypeConverterDelegate.java:294)
	at org.springframework.beans.AbstractNestablePropertyAccessor.convertIfNecessary(AbstractNestablePropertyAccessor.java:572)
	... 54 more
```